lsp-mode.el

```diff
@@ -650,6 +650,26 @@ If this is set to nil, `eldoc' will show only the symbol information."
   :type 'boolean
   :group 'lsp-mode)
 
+(defcustom lsp-modeline-code-actions-enable nil
+  "Wheter to show code actions on modeline."
+  :type 'boolean
+  :group 'lsp-mode)
+
+(defcustom lsp-modeline-code-actions-delay 0.2
+  "Number of seconds to wait before checking for code actions to show on modeline."
+  :type 'number
+  :group 'lsp-mode)
+
+(defcustom lsp-modeline-code-actions-kind-regex "quickfix.*\\|refactor.*"
+  "Regex for the code actions kinds to show in the modeline."
+  :type 'string
+  :group 'lsp-mode)
+
+(defface lsp-modeline-code-actions-face
+  '((t :foreground "cyan2"))
```

Comment:
I guess it will be better to inherit `success` or another face from the core face? Otherwise, this might look bad in different themes?

Reply:
Good idea, changed to use emacs face `homoglyph`